diff -urN namazu-2.0.13.org/configure.in namazu-2.0.13/configure.in
--- namazu-2.0.13.org/configure.in	2004-05-21 01:34:35.761418776 +0200
+++ namazu-2.0.13/configure.in	2004-05-21 01:37:48.254155432 +0200
@@ -154,46 +154,10 @@
 dnl *
 dnl *  determine hostname and domainname
 dnl *
+hostname=localhost
+domainname=localdomain
 
-changequote(<<, >>)
-#
-#   determine hostname and domainname
-#
-hostname="`uname -n 2>/dev/null |\
-           awk '{ printf("%s", $1); }'`"
-if test "x$hostname" = "x"; then
-    hostname="`hostname 2>/dev/null |\
-               awk '{ printf("%s", $1); }'`"
-    if test "x$hostname" = "x"; then
-        hostname='unknown'
-    fi
-fi
-case $hostname in
-    *.* )
-        domainname=".`echo $hostname | cut -d. -f2-`"
-        hostname="`echo $hostname | cut -d. -f1`"
-        ;;
-esac
-if test "x$domainname" = "x"; then
-    if test -f /etc/resolv.conf; then
-        domainname="`egrep '^[ 	]*domain' /etc/resolv.conf | head -1 |\
-                     sed -e 's/.*domain//' \
-                         -e 's/^[ 	]*//' -e 's/^ *//' -e 's/^	*//' \
-                         -e 's/^\.//' -e 's/^/./' |\
-                     awk '{ printf("%s", $1); }'`"
-        if test "x$domainname" = "x"; then
-            domainname="`egrep '^[ 	]*search' /etc/resolv.conf | head -1 |\
-                         sed -e 's/.*search//' \
-                             -e 's/^[ 	]*//' -e 's/^ *//' -e 's/^	*//' \
-                             -e 's/ .*//' -e 's/	.*//' \
-                             -e 's/^\.//' -e 's/^/./' |\
-                         awk '{ printf("%s", $1); }'`"
-        fi
-    fi
-fi
-changequote([, ])
-
-admin="webmaster@${hostname}${domainname}"
+admin="webmaster@OOPS-YOU-SHOULD-PUT-REAL-WEBMASTER-EMAIL-IN-YOUR-mknmzrc-BEFORE-INDEXING"
 AC_MSG_CHECKING(for admin's email address)
 AC_MSG_RESULT($admin)
 OPT_ADMIN_EMAIL=$admin
